custom_components/tuya_local/devices/zemismart_spm02v25_energymeter.yaml

```diff
@@ -0,0 +1,196 @@
+name: Energy meter
+products:
+  - id: iwn0gpzz
+    manufacturer: Zemismart
+    model: "SPM02 Zigbee V2.5"
+entities:
+  - entity: sensor
+    name: Total Energy
+    class: energy
+    state_class: total_increasing
+    icon: mdi:home-import-outline
+    dps:
+      - id: 1
+        type: integer
+        name: sensor
+        unit: kWh
+        mapping:
+          - scale: 100
+
+  - entity: sensor
+    name: Phase A Raw Data
```

Comment:
This is not useful as a sensor, make it an attribute if you need to keep it, but most likely the format is the same as other devices that have phase A info on dp 6 and the data can be fully extracted into voltage, current and power readings, making the raw value redundant.